docs/monoDrive_home/Vehicle-Configuration.md

```diff
@@ -1,89 +1,213 @@
-# Vehicles
+# Scenario Vehicle
+The monoDrive Simulator provides users with the ability to customize the color and model to use for the ego vehicle as well as for the vehicles that interact with it on simulation. The simulator also provides users with the ability to specify the initial conditions for any car on the scene, i.e. initial pose and initial velocity. 
 
-## Vehicle
+## Change the vehicle's model
+<div class="img_container">
+  <video width=650px height=480px muted controls autoplay loop>
+    <source src="http://cdn.monodrive.io/readthedocs/vehicles.mp4" type="video/mp4">
+  </video>
+</div>
+**Current Blueprints available**   
+On the monoDrive Editor look for the Content Browser, from the folders select Vehicles. You will find the current models available.
+
+When selecting a model programmatically (i.e. using any client) you can assign the model to use for the ego vehicle to be random from the vehicle blueprints, you can find it on your Editor folder under:  
```

Comment:
Try this:  When selecting a model programmatically (i.e. using any client) the user can assign the model for the ego vehicle randomly from the vehicle blueprints. This can be found in the `Editor folder`